web/app/components/comment/comment.jsx

```diff
@@ -286,6 +286,7 @@ export default class Comment extends Component {
                 <a
                   className="comment__link-to-parent"
                   href={`${o.locator.url}#${COMMENT_NODE_CLASSNAME_PREFIX}${o.pid}`}
+                  aria-label="Parent comment"
```

Comment:
`Go to parent comment`? And maybe it's better to add `title` here too.

Reply:
Agreed. Did that just for conciseness, our screen readers tend to say quite a lot of things at once :).